### DISCUSSION TITLE
CVE-2020-36518 Patch

### DISCUSSION THREAD

**Discussion by mj-easy** [playframework/playframework#11425] [General]
I was going to submit a patch for [CVE-2020-36518](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-36518) but it appears someone patched it already in https://github.com/playframework/playframework/pull/11351. Is there an ETA for Play 2.18.17 at the moment? 

Regards.

**Comment by mkurz** [comment]
Play 2.8.x still uses Jackson 2.11.4: https://github.com/playframework/playframework/blob/48c7456621fef948a1df5b3857f115c659223a2c/project/Dependencies.scala#L39-L40
The pull request you refer is for the main branch.

**Reply by mj-easy** [reply]
Oh! I see.
